mlflow/store/tracking/rest_store.py

```
@@ -445,15 +447,36 @@ def search_traces(
         sql_warehouse_id: Optional[str] = None,
     ):
         if sql_warehouse_id is None:
-            request = SearchTraces(
-                experiment_ids=experiment_ids,
+            # Create trace_locations from experiment_ids for the V3 API
+            trace_locations = []
+            for exp_id in experiment_ids:
+                try:
+                    location = TraceLocation.from_experiment_id(exp_id)
+                    proto_location = location.to_proto()
+                    trace_locations.append(proto_location)
+                except Exception as e:
+                    _logger.error(f"Error creating location for experiment ID {exp_id}: {e!s}")
```

Can we raise an exception here? Most of the case the given experiment ID is invalid format, then we should raise it to users rather than return incomplete search results.

---

Done

---

Do we still need `_logger.error()` call?